backend/src/main/java/ch/puzzle/pcts/service/validation/ValidationConfig.java

```diff
@@ -0,0 +1,36 @@
+// package ch.puzzle.pcts.service.validation;
```

Comment:
@Miguel7373  remove this file we do not need any special config. The spring native way works just fine for our usecase.